test/controllers/actions/ValidateMessageActionSpec.scala

```diff
@@ -306,7 +353,7 @@ class ValidateMessageTypeActionSpec
       val exampleRequest: JsValue = Json.parse(
         """{
           |     "message": {
-          |         "messageType": "IE028"
+          |         "messageType": "IE005"
```

Comment:
Is IE028 not good enough?

Reply:
IE028 is now considered a valid message 😄